src/mlpack/methods/reinforcement_learning/q_learning_impl.hpp

```diff
@@ -184,6 +184,18 @@ void QLearning<
   updatePolicy->Update(learningNetwork.Parameters(), config.StepSize(),
       gradients);
   #endif
+
+  if (config.NoisyQLearning() == true)
+  {
+    learningNetwork.ResetNoise();
+    targetNetwork.ResetNoise();
+  }
+  // Update target network
```

Comment:
```suggestion
  // Update target network.
```

Missing `.` at the end.